modules/tinymce/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Binding an event handler incorrectly took effect immediately while the event was firing #TINY-7436
 - Partially transparent RGBA values provided in the `color_map` setting were given the wrong hex value #TINY-7163
 - Fixed an issue when pasting cells from tables containing `colgroup`s into tables without `colgroup`s #TINY-6675
+- Word count was not updated when editor content has been changed by remote part #TINY-7168
```

Comment:
This is for RTC so we shouldn't have a changelog or types for this yet.
```suggestion
```

Reply:
A rule of thumb is if the PR is not changing editor behavior or editor apis then there is no need for a changelog item. For example internal refactoring to get rtc working better is only effecting rtc users like this case. But it's a bit of a gray area since we are adding a event but it will only be emitted when the rtc plugin is enabled.

Reply:
removed changelog update